README.md

```diff
@@ -0,0 +1,65 @@
+# Cloudblue Connect Mulesoft Extension
+
+## Overview
+CloudBlue Connect is a supply automation platform that between others manages products, services, contracts, ordering and fulfillment, usage and subscription.
+It supports any product, from physical goods to cloud products, as well as any channel, including your direct and indirect sales channels and internal procurement.
+Vendors can leverage CloudBlue Connect to:
+-   Reduce the total cost of ownership for homegrown technology supporting their indirect channel
+-   Standardize integrations with partners
+-   Increase efficiencies and minimize redundancies by bridging their direct and indirect sales channels
+
+Distributors can use CloudBlue Connect to:
+-   Define, manage and distribute any type of product (omni-product) through any channel (omni-channel)
+-   Transform perpetual licensed products into a subscription model
+-   Onboard new products into their portfolio quickly to build and deliver unique solutions to end customers
+
+Mulesoft Extension for Cloudblue Connect enables you to create apps that react to Cloudblue Connect events such as adding, changing, or deleting objects like purchase requests, assets, accounts.
+
+It allows an easy and fast integration with the CloudBlue Connect Platform. Thanks to it you can automate the fulfillment of orders generated by your products.
+
+Cloudblue Connect Mulesoft Extension exposes methods for accessing Cloudblue Connect and uses the public API defined at [Cloudblue Connect OpenAPI Specification](https://connect.cloudblue.com/community/api/openapi/).
+
+
+This extension has all of the functionallity exposed in the API definition, and provides you with all access to the functionality of the Connect Platform.
+
+## Prerequisites
+To use this connector, you must be familiar with:
+
+-   The Cloudblue Connect operation    
+-   Anypoint Studio    
+-   Mule runtime engine (Mule)   
+-   Elements and global elements in a Mule flow    
+-   Creating a Mule app using Anypoint Studio (Studio)
+
+## Getting Started
+
+In this repository you will find the code to deploy the Cloudblue Connect Mulesoft Extension. Also you can download from Maven repository. 
```

Comment:
The code in this repo is not recommended for self build and deploy. Better to guide the viewer to only Maven Repository.
Also include dependency code sample to include in pom.xml

Reply:
If you want to install in Anypoint Exchange you must download the repo, modify the pom.xml with the specific data from the user and deploy... then both options are possibles.